core/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -94,6 +94,10 @@ abstract class AppBase(
   var sparkRapidsBuildInfo: SparkRapidsBuildInfoEvent = SparkRapidsBuildInfoEvent(immutable.Map(),
     immutable.Map(), immutable.Map(), immutable.Map())
 
+  // core millisec (per executor) is computed as executor duration (ms) multiplied by num of cores
+  // totalCoreMs is the sum of all executor core millisecs
+  var totalCoreMs: Long = 0L
```

Comment:
I don't see the value of storing totalCoreMs as a variable as it is calculated after parsing the eventlog is complete and there is no dependency on that value.

Reply:
updated this, thanks!